README.md

```diff
@@ -150,7 +151,7 @@ git clone https://github.com/django/django.git
 Enter the command:
 
 ```sh
-sudo pip install django
+pip install django
```

Comment:
Thanks! Shouldn't the installation be without OS' then if we end up using the same command?

Reply:
Yes. I don't know how much you want to go into the details, but if you use virtualenvwrapper or virtualenv in general, syntax is pretty much the same. No need to use sudo since everything is being installed in local spaces of virtualenv.

Reply:
Could you fix that and also add yourself to the authors, I'll then merge your pr 😃